pinot-common/src/main/java/org/apache/pinot/common/config/TableNameBuilder.java

```diff
@@ -113,6 +114,26 @@ public static String extractRawTableName(@Nonnull String tableName) {
     return tableName;
   }
 
+  /**
+   * Get the table name after adding the given type suffix.
+   *
+   * @param tableNameOptType Table name with or without type suffix
+   * @param tableType Table type suffix
+   * @return Table name with type suffix
+   */
+  public static String getTableNameWithType(String tableNameOptType, @Nullable CommonConstants.Helix.TableType tableType) {
```

Comment:
You can use `TableNameBuilder.forType(tableType).tableNameWithType(tableName)` which IMO is more readable

Reply:
Thanks for the suggestion. I saw it in TableViews.java and moved it here. I was going to use it but later found other alternatives.